.github/workflows/wheels-manylinux-test.yml

```diff
@@ -90,12 +90,18 @@ jobs:
           export MATRICES="
             pull-request:
               - { arch: 'amd64', python: '3.9', ctk: '11.8.0', image: 'ubuntu18.04', test-type: 'unit', test-command: ${UNIT_TEST_CMD}, gpu: 'v100', driver: 'latest' }
+              - { arch: 'amd64', python: '3.9', ctk: '12.0.1', image: 'ubuntu18.04', test-type: 'unit', test-command: ${UNIT_TEST_CMD}, gpu: 'v100', driver: 'latest' }
               - { arch: 'arm64', python: '3.9', ctk: '11.8.0', image: 'ubuntu20.04', test-type: 'smoke', test-command: ${SMOKE_TEST_CMD}, gpu: 'a100', driver: 'latest' }
+              - { arch: 'arm64', python: '3.9', ctk: '12.0.1', image: 'ubuntu20.04', test-type: 'smoke', test-command: ${SMOKE_TEST_CMD}, gpu: 'a100', driver: 'latest' }
```

Comment:
I am going to open a follow-up PR to remove the `arm64` testing for wheels from `11.8` now that `12.0` is out.

We have very limited (8) `arm` GPU runners.

With this `arm` addition, a single PR will take up 4 GPUs:

- 1 arm GPU for conda C++ tests
- 1 arm GPU for conda Python tests
- 2 arm GPUs for wheel tests

This will cause really long queues for the arm GPU runners.

Reply:
Can we do smoke tests instead of full tests?

Reply:
That's what it's currently doing, but I still think it's impractical to have a single PR take up half of our ARM GPUs.

I think it's a fine trade-off to test only the latest CUDA version for ARM in PRs and rely on nightly testing for additional confidence.

We can always widen this matrix again in the future when we get more capacity.